cmd/root.go

```diff
@@ -34,6 +36,12 @@ var (
 	// Is guaranteed to be set and valid after InitAssetCmds is called (if it is not valid, an error is returned and
 	// the program will not run).
 	assetRunner *GolangCILintAssetRunner
+
+	// Package-level variables set by InitConfig.
+	// Is guaranteed to be set and valid after InitConfig is called.
+	// Note that pluginConfig may be nil if the plugin configuration file is empty (and this is a valid state).
+	excludes     matcher.NamesPathsCfg
+	pluginConfig *config.PluginConfig
```

Comment:
Was there any particular reason why you chose pkg level vars instead of potentially returning a struct from `InitConfig()` that includes this information which could be passed to `InitAssetCmds()`?

Reply:
Thanks for flagging this -- I took a closer look, and actually there's not.

This was a holdover from the `okgo` plugin -- for that plugin, assets actually contributed directly to the available `cobra` commands, so it was necessary to load the asset information before invoking the cobra command tree.

However, the current structure of the plugins and assets does not require this, so I've refactored the logic to get rid of the package-level variables.